docs/docs/machine-setup.md

```diff
@@ -1,112 +1,64 @@
+    2. Create the VM.
+    ```bash
+    az vm create \
+    --size Standard_DC4s_v3 \
+    --image Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:latest \
+    --name <VM-NAME> \
+    --resource-group <GROUP-NAME>
+    ```
 
+    `VM-NAME` is the name you choose for the VM, and `GROUP-NAME` is the name of your resource group.
 
-## Hardware support
+=== "Azure Portal"
 
-The machine needs support from both the CPU and firmware. Please check with your hardware manufacture to confirm that your machine supports SGX.
+    Make sure you do the following when creating your Azure Confidential Computing VM:
 
-For some machines SGX must be explicitly enabled in the BIOS/UEFI firmware screens. For others, it can be activated
-by any root user: the Conclave host API will try to activate it for you, if possible and if run with sufficient
-permissions.
+    * Use the `Ubuntu Server 20.04 (Gen2)` image
+    * Pick a size that's got plenty of RAM. You might have to click "Change size" to find the `DC4s_v3` type
+    * Ensure that the public inbound ports are open for SSH access
 
-## Hosting providers
+    ![](images/create-dc4sv3-vm.png)
+    ![](images/create-d4sv3-vm-publicinbound.png)
 
-* Microsoft Azure offers [virtual machines with SGX hardware](https://azure.microsoft.com/en-us/solutions/confidential-compute/)
-* OVH offers [rentable SGX hardware](https://www.ovhcloud.com/en-gb/bare-metal/intel-software-guard-extensions/)
+After you have logged on to the VM:
 
-## Distribution support
+1. Check that the `enclave` device is in the `/dev/sgx/` or `/dev/sgx_*` directory.
+2. Add your user into the `sgx_prv` group to give it access to SGX.
+```sh
+sudo usermod -aG sgx_prv $USER
+```
+3. Log out from the VM and log in again.
 
-The following Linux distros are formally supported by Intel:
+You have set up an Azure VM with the latest attestation protocols.
 
-* Ubuntu 18.04 LTS Desktop 64bits
-* Ubuntu 18.04 LTS Server 64bits
-* Ubuntu 20.04 LTS Desktop 64bits
-* Ubuntu 20.04 LTS Server 64bits
-* Red Hat Enterprise Linux Server release 7.6 64bits
-* Red Hat Enterprise Linux Server release 8.2 64bits
-* CentOS 8.2 64bits
-* Fedora 31 Server 64bits
+You need to use the DCAP protocol for attestation. For this, you need to use the
+[`AttestationParameters.DCAP`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.host/-attestation-parameters/-d-c-a-p/-attestation-parameters.-d-c-a-p.html) class when starting the
+enclave via
+[`EnclaveHost.start`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.host/-enclave-host/start.html).
 
-However, others will probably still work.
+!!!Important
```

Comment:
I feel that this is a bit out of context here. Here above we actually explained how to set up a VM in Azure for Conclave, we haven't really talked about deployment.
But this `Important` section is related to issues when deploying a Conclave app and what happens on restarts, which is not really related to installation but to usage.
Maybe we can just explain better:
```
When using a Conclave application on Azure, you should remember that Azure does not guarantee access to the same machine on reboot. As encrypted secrets are strictly tightened to the Intel SGX CPU running the VM, you might lose them if Azure moves the VM to a different machine. To prevent...```

Reply:
Done.